config/config.yaml

```diff
@@ -149,3 +155,16 @@ cluster:
 # Set to true to prevent service from sending usage statistics to the developers.
 # Read more: https://qdrant.tech/documentation/telemetry
 telemetry_disabled: false
+
+
+# TLS settings
+tls_config:
+
+  # Server certificate chain file
+  cert: /tls_path/cert.pem
+
+  # Server private key file
+  key: /tls_path/key.pem
+
+  # Client certificate file (required if cluster.p2p.enable_tls = true)
+  client_cert: /tls_path/cert.pem
```

Comment:
I guess, I have two comments regarding the configuration (that I also don't have the best answers to, so @generall):

- `cert` and `client_cert` parameter names are a bit confusing, IMO
  - e.g., `client_cert` is not a certificate that the client will be presenting to the server, but the CA that the client will be using to validate the server
- `tls_config` section is not optional
  - and `cert`/`key`/`client_cert` keys aren't optional either
  - so all of them will have to be present and defined to _something_, even if TLS is disabled

Reply:
I would also change the default folder to something like `./tls`, so we could run it without docker as well (and in docker it would be `/qdrant/tls`

Reply:
Also changed accordingly.